src/main/java/org/folio/des/scheduling/bursar/BursarScheduledJobInitializer.java

```diff
@@ -0,0 +1,34 @@
+package org.folio.des.scheduling.bursar;
+
+import java.util.Optional;
+
+import org.folio.des.domain.dto.ExportConfig;
+import org.folio.des.scheduling.BursarExportScheduler;
+import org.folio.des.service.config.ExportConfigService;
+import org.springframework.stereotype.Component;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+
+@Component
+@RequiredArgsConstructor
+@Log4j2
+public class BursarScheduledJobInitializer {
+
+  private final ExportConfigService burSarExportConfigService;
+  private final BursarExportScheduler bursarExportScheduler;
+
+  public void initAllScheduledJob() {
+    log.info("initiating scheduled job of type Bursar");
+    try {
+      Optional<ExportConfig> savedConfig = burSarExportConfigService.getFirstConfig();
+      if (savedConfig.isPresent()) {
+        bursarExportScheduler.scheduleBursarJob(savedConfig.get());
+      } else {
+        log.info("No export schedules found.");
+      }
+    } catch (Exception e) {
+      log.error("get configuration failed for type BURSAR");
```

Comment:
Exception might be not only in getting configuration, but from scheduling as well. also it looks more like "warn" level